docker_files/init.pl

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use lib qw(/app /opt/bmo/local/lib/perl5);
+use Getopt::Long qw(:config gnu_getopt);
+use Data::Dumper;
+use Bugzilla::Install::Localconfig ();
+use Bugzilla::Install::Util qw(install_string);
+
+my %localconfig = (webservergroup => 'app');
+
+my %override = (
+    'inbound_proxies'     => 1,
+    'memcached_namespace' => 1,
+    'memcached_servers'   => 1,
+    'shadowdb'            => 1,
+    'shadowdbhost'        => 1,
+    'shadowdbport'        => 1,
+    'shadowdbsock'        => 1
```

Comment:
So some settings are... backwards, and stored in data/params. Only those backwards setting get the 'override' treatment. apache_size_limit is a localconfig option and thus needs no special voo-doo. Those settings should have never been in data/params to begin with.